src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -148,12 +148,12 @@ protected override bool TrySerializeKnownTypes(object input, out object output)
             }
 
             var inputType = input.GetType().GetTypeInfo();
-            if (this.TrySerializeJavaScriptConverter(input, out output, inputType))

```

Comment:
You have `var inputType = input.GetType().GetTypeInfo();` above, can't you use that? :stuck_out_tongue: